index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
+        </div>
+      </div>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+          <h5 class="card-title">API - Google Tag Manager</h5>
+          <ul>
+            <li class="badge text-bg-secondary">
+              Google Cloud Platform Services
+            </li>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">Google Tag Manager API</li>
+          </ul>
+          <p class="card-text">
+            ExpressJS API used for the automated deployment of Google Tag
+            Manager servers to Google Cloud Platform.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
+        </div>
+      </div>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+          <h5 class="card-title">
+            API - MySQL data to Facebook Conversions API
+          </h5>
+          <ul>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">MySQL</li>
+            <li class="badge text-bg-secondary">Facebook Conversions API</li>
+          </ul>
+          <p class="card-text">
+            ExpressJS API that imports data from a MySQL database on Azure and
+            sends Conversions API events in real time from their customer
+            interactions that cannot be tracked on the website.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="d-flex flex-column m-4 p-2 about-me">
+      <h1 class="primary py-2 my-2 text-center">About me</h1>
+      <p>
+        I can help you build a product, feature or website. If you like what you
+        see, don't hesitate to contact me.
+      </p>
+      <a href="#" class="btn btn-primary teal-button">Get my resume</a>
+
+      <span class="badge badge-pill badge-primary">Primary</span>
+      <div class="card my-2 text-center skills">
+        <div class="card-body justify-content-center">
+          <img src="icons/icon-languages.svg" alt="languages" />
+          <h5 class="card-title">Languages</h5>
+          <ul>
+            <li class="badge text-bg-secondary">Typescript</li>
+            <li class="badge text-bg-secondary">Javascript</li>
+            <li class="badge text-bg-secondary">HTML/SASS</li>
+            <li class="badge text-bg-secondary">Ruby</li>
+            <li class="badge text-bg-secondary">C++</li>
+            <li class="badge text-bg-secondary">MySQL</li>
```

Comment:
- [ ] KIndly make your tech stack have the background color as the one on the provided Figma design. :wink:
- [ ] Kindly make sure the technologies list doesn't go out the card.

   - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/200127791-9e43edf0-d0f9-482c-b02e-6d69bea09bb0.png)


   - ### FIGMA DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/200127811-b78fc723-e8bb-4709-b4e5-48b15000ab74.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/44822765/200138520-170a2bb6-c424-43a0-8669-aaff6a14e06c.png)